File: program/program/print-fibonacci-series-without-recursion/PrintFibonacciSeriesWithoutRecursion.js
const number = 5

function fibonacci(num) {
    let fib = [0, 1];
    let data = [];
    for(let i = 2; i <= num; i++) {
      fib[i] = fib[i - 1] + fib[i - 2]; 
      data.push(fib[i]);
    }
    return [0, 1].concat(data)
}

console.log("Number : ", number);
console.log("Output : ", fibonacci(number));

File: program/program/print-fibonacci-series-without-recursion/PrintFibonacciSeriesWithoutRecursion.swift
import Foundation
import Glibc

func fibonacciSeries(num: Int) -> Int{
   // The value of 0th and 1st number of the fibonacci series are 0 and 1
   var n1 = 0
   var n2 = 1

   // To store the result
   var nR = 0
   // Adding two previous numbers to find ith number of the series
   for _ in 0..<num{
      nR = n1 
      n1 = n2
      n2 = nR + n2
   }
   return n1
}

var val = 10

// Iterate for every number stating from 0 to val
for j in 0...val{
   let output = fibonacciSeries(num: j)
   print(output)
} 
